### DISCUSSION TITLE
Wow new updates

### DISCUSSION THREAD

**Discussion by sindizzy** [DotSpatial/DotSpatial#1456] [General]
I am high in the sky for all the new activity going on with DotSpatial. :)

Let me ask this...if I were to start doing some PRs would I do it against v4? 
Is that version only for core projects or can framework use the libraries as well?

**Comment by jany-tenaj** [comment]
Huh? Not sure what you're asking, but if you have something to add then it goes into the master branch which targets .net 6.

**Comment by sindizzy** [comment]
I saw updates to v2 then a v3 and then a v4 so i guess Im way behind on releases. 
If I add a PR to the master branch then I'm assuming that's based on v4 right?
And those v4 libraries...they are based on Core. Can they be used on a Framework project?
